apps/frontend/src/libs/components/button/styles.module.css

```diff
@@ -14,6 +14,11 @@
 	outline: 0;
 }
 
+.btn:disabled {
```

Comment:
In my PR #430 I already updated disabled state for primary and secondary buttons